### PR TITLE
Ut for afd formfields

### PR DESCRIPTION
**Pull request by garimaprachi**

### CODE REVIEW

File: tests/phpunit/tests/test-afd-formfields.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    public function testinitformfields()
+    {
+        $GLOBALS['current_screen'] = new Globals();
+
+        $this->instance->shouldReceive('getRazorpayApiInstance')->andReturnUsing(function () {
+            return new MockApi('key_id_2', 'key_secret2');
+        });
+
+        $this->instance->init_form_fields();
```
Comment (yashgit891) [inline tests/phpunit/tests/test-afd-formfields.php:29]: Write test when feature is enabled.

Comment (garimaprachi) [reply tests/phpunit/tests/test-afd-formfields.php:29]: Done

File: tests/phpunit/mockfactory/Request.php [hunk 2]
```diff
@@ -174,6 +174,14 @@ public function loadData()
             ],
             'key_id_2' => [
                 'GET' => [
+                    'accounts/me/features' =>
+                        [
+                            'assigned_features' =>[
```
Comment (abdulwahidsharief) [inline tests/phpunit/mockfactory/Request.php:179]: @garimaprachi leave space after =>

Comment (garimaprachi) [reply tests/phpunit/mockfactory/Request.php]: Done

**Review by abdulwahidsharief — APPROVED** [review]
LGTM

**Pull request merged**